GoogleDataTransport/GDTCORLibrary/GDTCORRegistrar.m

```diff
@@ -19,6 +19,10 @@
 
 #import "GDTCORLibrary/Public/GDTCORConsoleLogger.h"
 
+id<GDTCORStorageProtocol> _Nullable GDTCORStorageInstanceForTarget(GDTCORTarget target) {
```

Comment:
Just curiosity: what are advantages of using a global C function vs a static method on e.g. `GDTCORRegistrar` for this case?

Reply:
The GDTCORRegistrar class doesn't expose any other internals, and since exposing the storage instances is only a thing needed as part of using an id<GDTCORStorageProtocol> it seemed to make sense to include it in that header. Just seemed like a more coherent API.

Reply:
Makes sense, thank you for explaining!